packages/next/src/build/utils.ts

```diff
@@ -1140,14 +1142,21 @@ export type AppConfig = {
   fetchCache?: 'force-cache' | 'only-cache'
   preferredRegion?: string
 }
-export type GenerateParams = Array<{
+
+type Params = Record<string, string | string[]>
```

Comment:
Should this be `Param`? Each instance represents one param name it seems

Reply:
This is a `Record`, so it really represents _many_ parameters, such as:

```ts
{
  "slug": "hello-world",
  "author": "wyatt"
} satisfies Record<string, string | string[]>
```

Reply:
Oh I missed that. Se we have an array of records of params?